docs/ManualDeploymentGuideScalarDLAuditorOnAWS.md

```diff
@@ -52,35 +52,13 @@ so you need to add peering for internal communication between the Auditor, Ledge
 * You must create peering between Ledger, Auditor, and Client VPCs to enable communication between private networks.
 * You must update Ledger, Auditor, and Client route tables for VPC peering connections.
 
-### Recommendations
-
-* You should create Network ACLs for Ledger and Auditor VPCs.
-* You should set up the Network ACLs of the Ledger and Auditor to restrict unused accesses.
```

Comment:
@kota2and3kan san,
Yes, we can integrate this suggestion as a recommendation. 
In my understanding, the default security group of the EKS cluster will enable only Ledger and Auditor Envoy LoadBalancers in the security group.

This suggestion may be more useful for creating customized security groups.

I will create another PR for integrating these changes.

Could you please correct me, If my understanding is wrong?

Reply:
Thank you for comfirmation!

Yes. Your understanding is correct.
I think it is better to describe information for customizing. 
It is more useful for users who want to customize their environment based on their requirement (e.g. Their own security rules).

Also, the security configuration is depends on the user's reqirement and it is resposibility of users, but it is recommended to restrict non-use connections in the production, because Scalar DL treats the sensitvie data. 
I think this is our (Scalar's) recommendation. 
To tell users our recommendation clearly, I think it is better to state (remain) this security recommendation.

Reply:
@kota2and3kan san,
Thank you for your confirmation. 
I will create another PR for this.